Ghidra/Features/VersionTracking/src/main/java/ghidra/feature/vt/gui/actions/AutoVersionTrackingTask.java

```diff
@@ -222,7 +222,7 @@ private void doRun(TaskMonitor realMonitor) throws CancelledException {
 			options.setDouble(scoreOption, minCombinedReferenceCorrelatorScore);
 
 			monitor.setPrefix(String.format(prefix, "Data Reference", ++count));
-			hasApplyErrors = hasApplyErrors | correlateAndPossiblyApply(factory, options, monitor);
+			hasApplyErrors = hasApplyErrors || correlateAndPossiblyApply(factory, options, monitor);
```

Comment:
This is a change in behavior.  This code intends to run `correlateAndPossiblyApply()` regardless of the previous error state.  This could should have been written as:
```
hasApplyErrors |= correlateAndPossiblyApply(factory, options, monitor);
```

so that the state of the flag is maintained while also calling the correlate method.